pages/task/read/content/base.js

```diff
@@ -1,5 +1,16 @@
 module.exports = {
   title: 'Review Application',
+  status: {
+    title: 'Current status',
+    'with-licensing': 'With licensing',
+    'referred-to-inspector': 'Referred to inspector',
+    'returned-to-applicant': 'Returned to applicant',
+    'withdrawn-by-applicant': 'Withdrawn by applicant',
+    'with-ntco': 'With NTCO',
+    'ntco-endorsed': 'NTCO endorsed',
+    'inspector-recommended': 'Inspector recommended for approval',
+    'inspector-rejected': 'Inspector recommended for rejected'
+  },
```

Comment:
I think this list now exists in a number of different places, and not necessarily in the same form. Can we consolidate?

Reply:
yes, I did have a look in asl-constants but couldn't find it

Reply:
I guess it just needs to be in the base content for all tasks, rather than asl-constants

Reply:
It's not in constants, I meant various content files in `pages/task/*`

Reply:
https://github.com/UKHomeOffice/asl-pages/blob/master/pages/task/read/content/success.js#L17-L41

Reply:
Also here: https://github.com/UKHomeOffice/asl-pages/blob/master/pages/task/list/content/index.js#L26-L36